doc/02_Installation/README.md

```diff
@@ -84,6 +84,10 @@ Following options to prepare the customer class are available:
   [`CustomerManagementFrameworkBundle\Model\AbstractCustomer\DefaultAbstractUserawareCustomer`](https://github.com/pimcore/customer-data-framework/blob/master/src/Model/AbstractCustomer/DefaultAbstractUserawareCustomer.php) 
   class and also need to have one additional data attribute:
   * `password`: password field
+
+  Note: When extending the customer class from `DefaultAbstractUserawareCustomer` then either Customer class adds the required attributes for 
+ getters/setters defined in the ['CustomerManagementFrameworkBundle\Model\CustomerInterface'](https://github.com/pimcore/customer-data-framework/blob/master/src/Model/CustomerInterface.php)
+    or Use a Trait, that implements these getters/setters, in Pimcore class.
```

Comment:
Which Trait?

Reply:
A custom trait with getters/setters

Reply:
Where can I find it? The documentation here does not make it more clearer from my point of view, it just looks like "outsourcing" by means of trial-and-error rather than an actual clarification or fix.

Reply:
Added a gist link for Trait implementation.

Reply:
If this is the default anyone should use, why not include it in the Abstract class anyway?